README.md

```diff
@@ -34,23 +53,122 @@ They are said to be "bound" to the unit they are currently operating on.
 
 | Abbreviation | Description | Implementation State |
 | ---          | ---         | ---                  |
-| cut          | Cut Strand(s) | DONE! |
-| del          | Delete a base from strand | |
-| swi          | Switch enzyme to other strand | |
-| mvr          | Move one unit to the right | |
-| mvl          | Move one unit to the left | |
-| cop          | Turn on copy mode | |
-| off          | Turn off copy mode | |
-| ina          | Insert A to the right of this unit | |
-| inc          | Insert C to the right of this unit | |
-| ing          | Insert G to the right of this unit | |
-| int          | Insert T to the right of this unit | |
-| rpy          | Search of the nearest pyrimidine to the right | |
-| rpu          | Search for the nearest purine to the right | |
-| lpy          | Search for the nearest pyrimidine to the left | |
-| lpu          | Search for the nearest purine to the left   | |
-
-### Binding Preference
+| [pun](#pun)  | Punctuates strands, allowing them to encode for multiple enzymes | DONE! |
+| [cut](#cut)  | Cut Strand(s) | |
+| [del](#del)  | Delete a base from strand | |
+| [swi](#swi)  | Switch enzyme to other strand | |
+| [mvr](#mvr)  | Move one unit to the right | |
+| [mvl](#mvl)  | Move one unit to the left | |
+| [cop](#cop)  | Turn on copy mode | |
+| [off](#off)  | Turn off copy mode | |
+| [ina](#ina)  | Insert A to the right of this unit | |
+| [inc](#inc)  | Insert C to the right of this unit | |
+| [ing](#ing)  | Insert G to the right of this unit | |
+| [int](#int)  | Insert T to the right of this unit | |
+| [rpy](#rpy)  | Search of the nearest pyrimidine to the right | |
+| [rpu](#rpu)  | Search for the nearest purine to the right | |
+| [lpy](#lpy)  | Search for the nearest pyrimidine to the left | |
+| [lpu](#lpu)  | Search for the nearest purine to the left   | |
+
+#### pun
+
+Not an Amino acid/instruction, but deserves special clarification. 
+
+Punctuates the strand, allowing one strand to encode for two or more separate enzymes. 
+When determining the enzymes encoded for by a strand, the prescence of the pun instruction 
+ends the enzyme early, and an additional enzyme is determined from the remaining strand, decoding 
+until the end of the strand or another pun is reached. These enzymes will act sequentially on
+the strand, one at a time. The portion of a strand that encodes for a specific enzyme is known
+as a 'gene', so one strand may contain one or more genes that encode for the same number of enzymes. 
```

Comment:
> Isn't it possible for a strand to have zero genes if it only consists of `AA`?

Yes, that's correct. I misunderstood the idea behind the AA base pair. It's not an amino acid, and it does not produce an enzyme. 


> Also, if a strand produces two enzymes, do they really process the entire strand? Not just the gene they were created from?

This is actually ambiguous, since no demonstration of 2 separate enzymes is shown. That was my big change in understanding behind punctuation when I first start implementing it. See [Typogenetics](https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=a512a815e6d7cd8a3c87c9997802ec35f0505bf7) , page 10. (As an aside, I have not viewed any of the code or examples in that paper, just found that looking for rules clarifications). So we should mention that this is the execution we chose in the Authors' notes.